port : "/dev/ttyS2"
baud : 115200

# specify robot track width in meters
track_width: 0.9925
# specify reduction ratio (70:1), if there is no put 1
reduction_ratio: 70
# specify reduction maximum linear velocity (m/s)
max_vel: 1  # 60 m/min @3500rpm
# specify robot wheel circumference (perimeter) in meters 
wheel_circumference: 1.162389 # D:37cm 

frequencyH : 50   #higher frequency (value is in ms)

queryH:
 motor_amps : ?A 
 motor_command : ?M
 hall_count : ?BCR
 abs_hall_count : ?CB
 hall_speed : ?BS
 volts : ?V
 feedback : ?F 
 battery_amps : ?BA
 power : ?P          
 fault_flag : ?FF  
# motor_amps2 : ?MA
# motor_command2 : ?FM
# hall_count2 : ?FM
# hall_speed2 : ?FS
# volts2 : ?V
# feedback2 : ?rmp 
# battery_amps2 : ?PHA
# power2 : ?sns          
# fault_flag2 : ?S  
# status_flag : ?FS
# firmware_id : ?FID  
# encoder_count : ?C
# encoder_speed : ?S  

